docs/source/version_compatibility.rst

```diff
@@ -0,0 +1,34 @@
+**************
+Sabre Versions
```

Comment:
Suggested title change: 

Sabre and Sawtooth Version Compatibility

... because this topic is about Sabre AND Sawtooth (and Docker, but the title doesn't need to include that).